core/templates/dev/head/player/AnswerClassificationServiceSpec.js

```diff
@@ -75,30 +88,9 @@ describe('Answer classification service', function() {
     }
   };
 
-  it('should query the backend when no frontend rules are provided',
-      function() {
-    var backendClassifiedOutcome = {
-      outcome: 'outcome',
-      answer_group_index: 0,
-      rule_spec_index: 0
-    };
-    var expectedClassificationResult = {
-      outcome: 'outcome',
-      answerGroupIndex: 0,
-      ruleSpecIndex: 0
-    };
-    $httpBackend.expectPOST(
-      '/explorehandler/classify/' + explorationId).respond(
-      backendClassifiedOutcome);
-    acs.getMatchingClassificationResult(explorationId, state, 0, false).
-      then(successHandler, failHandler);
-    $httpBackend.flush();
-
-    expect(successHandler).toHaveBeenCalledWith(expectedClassificationResult);
-    expect(failHandler).not.toHaveBeenCalled();
-
-    $httpBackend.verifyNoOutstandingExpectation();
-    $httpBackend.verifyNoOutstandingRequest();
+  it('should be undefined if no frontend rules are provided', function() {
+    expect(acs.getMatchingClassificationResult(explorationId, state, 0, false))
```

Comment:
Should this check that some alert should be thrown? In fact, should we throw an exception and break noisily since the learner won't be able to proceed?